samples/vanilla/index.html

```diff
@@ -0,0 +1,282 @@
+        const datetimePicker = document.querySelector('ef-datetime-picker');
+        const textArea = document.querySelector('textarea');
+        const emailField = document.querySelector('ef-email-field');
+        const menu = document.querySelector('ef-overlay-menu');
+        const jobPanel = document.querySelector('#job-panel');
+        const jobDisplay = document.querySelector('#job-display');
+        const checkbox = document.querySelector('ef-checkbox');
+        const confirmButton = document.querySelector('#confirm-button');
+        const cancelButton = document.querySelector('#cancel-button');
+        const chart = document.querySelector('ef-interactive-chart');
+
+        const data = [
+          {
+            'time': Date.parse('2019-04-10T08:40:10') / 1000,
+            'open': 141.77,
+            'high': 170.39,
+            'low': 120.25,
+            'close': 145.72,
+            'value': 145.32
+          },
+          {
+            'time': Date.parse('2019-04-10T09:40:10') / 1000,
+            'open': 145.72,
+            'high': 147.99,
+            'low': 100.11,
+            'close': 108.19,
+            'value': 124.05000000000001
+          },
+          {
+            'time': Date.parse('2019-04-10T10:40:10') / 1000,
+            'open': 108.19,
+            'high': 118.43,
+            'low': 74.22,
+            'close': 75.16,
+            'value': 96.325
+          },
+          {
+            'time': Date.parse('2019-04-10T11:40:10') / 1000,
+            'open': 75.16,
+            'high': 82.84,
+            'low': 36.16,
+            'close': 45.72,
+            'value': 59.5
+          },
+          {
+            'time': Date.parse('2019-04-10T12:40:10') / 1000,
+            'open': 45.72,
+            'high': 120.12,
+            'low': 44.1,
+            'close': 90.5,
+            'value': 82.11
+          },
+          {
+            'time': Date.parse('2019-04-10T13:40:10') / 1000,
+            'open': 90.5,
+            'high': 95.89,
+            'low': 75.58,
+            'close': 105.55,
+            'value': 85.735
+          }
+        ];
+        
+        // chart section
+        // initial chart config
+        chart.config = {
+          options: {
+            timeScale: {
+              timeVisible: true,
+              secondsVisible: true
+            }
+          },
+          series: [{
+            symbol: 'Price',
+            type: chartType,
+            data: data
+          }]
+        };
+
+        /* reset all tabs to inactive */
+        const resetActive = function () {
+          tabs.forEach(function (tab) {
+            tab.active = false;
+          });
+        };
+
+        tabs.forEach((tab) => {
+          const type = tab.label.substring(0, tab.label.indexOf(' ')).toLowerCase();
+          if (chartType === type) {
+            tab.active = true;
+          }
+          tab.addEventListener('tap', function (e) {
```

Comment:
The code has mixed function and arrow function. Can you make all function has the same code style?